staging/src/k8s.io/client-go/tools/clientcmd/api/helpers_test.go

```diff
@@ -208,11 +224,19 @@ func TestFlattenSuccess(t *testing.T) {
 
 func Example_minifyAndShorten() {
 	certFile, _ := os.CreateTemp("", "")
-	defer os.Remove(certFile.Name())
+
 	keyFile, _ := os.CreateTemp("", "")
-	defer os.Remove(keyFile.Name())
+
 	caFile, _ := os.CreateTemp("", "")
-	defer os.Remove(caFile.Name())
+
+	defer func() {
```

Comment:
why not use the `utiltesting.RemoveTestFile(t, f)` here?

Reply:
> why not use the `utiltesting.RemoveTestFile(t, f)` here?

this is an example function, it cannot accept parameters